### PR TITLE
[ADD] solution hw-6.3

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/java/core/basesyntax/FibonacciSequence.java [hunk 1]
```diff
@@ -10,6 +10,15 @@
...
-        return 0;
+        if (index == 0) {
+            return 0;
+        }
+        if (index == 1 || index == -1) {
+            return 1;
+        }
+        if (index < 0) {
+            return getFibonacciNumber(index + 2) + getFibonacciNumber(index + 1);
+        }
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FibonacciSequence.java]: why do we need to check for negative indices?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/FibonacciSequence.java (hunk 1 above)
Comment (ghost) [reply src/main/java/core/basesyntax/FibonacciSequence.java]: done

File: src/main/java/core/basesyntax/FibonacciSequence.java [hunk 2]
```diff
@@ -10,6 +10,12 @@
      * Результат: 21</p>
      */
     public int getFibonacciNumber(int index) {
-        return 0;
+        if (index == 0) {
+            return 0;
+        }
+        if (index == 1 || index == -1) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FibonacciSequence.java]: why do we need to check for negative indices?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/FibonacciSequence.java (hunk 2 above)
Comment (ghost) [reply src/main/java/core/basesyntax/FibonacciSequence.java]: done

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request closed without merging**